pyproject.toml

```diff
@@ -93,12 +93,18 @@ notebook-dependencies = [
 "Documentation" = "https://qiskit-extensions.github.io/circuit-knitting-toolbox/"
 "Repository" = "https://github.com/Qiskit-Extensions/circuit-knitting-toolbox"
 
+[tool.hatch.build.targets.wheel]
+only-include = [
+    "circuit_knitting",
```

Comment:
So users will have access to two namespaces, and one of the namespaces will throw a bunch of warnings?

Reply:
Exactly.